fuzz/fuzz.go

```diff
@@ -0,0 +1,116 @@
+package fuzz
+
+import (
+	"bufio"
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"time"
+)
+
+var failingInputRegex = regexp.MustCompile(`^\s*go test -run=Fuzz([a-zA-Z0-9_]+)/([a-zA-Z0-9#]+)`)
+var failingSeedInputRegex = regexp.MustCompile(`^\s*failure while testing seed corpus entry: Fuzz([a-zA-Z0-9_]+)/([a-zA-Z0-9#]+)`)
+
+type FailingInputError struct {
+	ID   string
+	File string
+	Seed bool
+}
+
+func (f FailingInputError) Error() string {
+	newOrSeed := "new"
+	if f.Seed {
+		newOrSeed = "seed"
+	}
+
+	if f.File != "" {
+		return fmt.Sprintf("failing %s input, saved at %s", newOrSeed, f.File)
+	}
+	return fmt.Sprintf("failing %s input: %s", newOrSeed, f.ID)
+}
+
+func (p *Project) relCorpusDir(target TestTarget) string {
+	// target.Package contains the root package as well
+	// we need to strip it because it refers to the current working directory .
+	pkg := strings.TrimPrefix(strings.TrimPrefix(target.Package, target.RootPackage), "/")
```

Comment:
Here maybe - for compatibility with Windows and more robustness - we could use [filepath.Rel](https://pkg.go.dev/path/filepath#Rel) and other methods from `filepath` (Clean, etc.).

(though we want to be sure we have the same output ... 😬 , as there is not unit tests around this one, might wait for another PR)